### PR TITLE
unit-tests-jest created - William Zhang

### PR DESCRIPTION
**Pull request by williamzhang73**
image

### CODE REVIEW

File: unit-tests-jest/numbers.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+ * Returns a number formatted in dollars and cents.
+ */
+export function toDollars(amount: number): string {
+  if (!Number.isInteger(amount)) {
+    return 'undefined';
+  }
+  if (amount < 0) {
+    return 'bigger than 0';
+  }
+  return `$${amount}.00`;
```
Comment (RobertGardner) [inline unit-tests-jest/numbers.ts]: This won't work correctly for fractional numbers.

**Review by RobertGardner — APPROVED** [review]
Some nice edge case tests in here!
